e2e/workloads/deploy.go

```diff
@@ -57,6 +57,14 @@ func (w Deployment) GetBranch() string {
 	return w.Branch
 }
 
+func (w Deployment) GetSelectResource() string {
+	return "deployment"
+}
+
+func (w Deployment) GetLabelName() string {
+	return "appname"
+}
```

Comment:
What about the value of label? are we assuming that it is always the name of the application (e.g. busybox)?